webapp/tests/elster_client/test_elster_client.py

```diff
@@ -107,27 +108,35 @@ def test_if_eric_transfer_error_occurred_raise_error(self):
         finally:
             MockErica.eric_transfer_error_occurred = False
 
-    def test_if_erica_timeout_error_occurred_raise_error(self):
-        MockErica.invalid_request_timeout_occurred = True
+    def test_if_erica_timeout_error_occurred_then_raise_corresponding_error(self):
```

Comment:
👍 Could you just put those three tests into their own class `TestSendToErica` to make sure that we keep tests for different methods in separate test classes? 🙈

Reply:
Yes. :heavy_check_mark: